src/resolvers/git.cr

```diff
@@ -104,7 +104,9 @@ module Shards
       case key
       when "git"
         {"git", source}
-      when "github", "bitbucket", "gitlab"
+      when "github"
+        {"git", "https://#{key}.com/#{source.downcase}.git"}
+      when "bitbucket", "gitlab"
```

Comment:
Project URLs on bitbucket and gitlab are case insensitive as well, so might apply this change to all of them.

Reply:
yeah I wasn't actually 100% on that, but just tested and seems to be the case!